src/main.py

```diff
@@ -263,6 +263,13 @@ def parse_flags(flags: List[str]) -> Options:
         default=[],
         help="Mark preprocessor constant as undefined",
     )
+    group.add_argument(
+        "--incbin-dir",
+        dest="incbin_dir",
+        default=None,
+        type=Path,
+        help="Search path for loading .incbin directives in the input asm",
```

Comment:
Would be nice to allow passing this multiple times. (I agree with not using -I for this for now, since it would conflict a bit with C context)